app/controllers/api/v1/sessions_controller.rb

```diff
@@ -32,13 +32,18 @@ def index
       # POST /api/v1/sessions
       # Signs a user in and updates the session cookie
       def create
-        user = User.find_by(email: session_params[:email], provider: current_provider)
-
         # TODO: Add proper error logging for non-verified token hcaptcha
         return render_error if hcaptcha_enabled? && !verify_hcaptcha(response: params[:token])
 
+        user = User.find_by(email: session_params[:email], provider: [current_provider, 'bn'])
```

Comment:
This should solve the issue if a user has multiples accounts in different tenants.
But, this will default to a 'bn' account if the user email is associated with one - which seems reasonable.

Reply:
This is not behaving as expected upon more testing.

Reply:
Nope. We should prioritize current provider, and if not found, it should then search for bn

Reply:
Will now initially search for a user within the current_provider. 
If not found, it will search for a user with bn provider.